frontend/pages/teams/[slug]/question/[question_slug].tsx

```diff
@@ -0,0 +1,212 @@
+import CommentForm from '@/components/organisms/CommentForm'
+import SortDropdown from '@/components/molecules/SortDropdown'
+import AnswerForm from '@/components/organisms/AnswerForm'
+import AnswerDetail from '@/components/organisms/AnswerDetail'
+import Comment from '@/components/organisms/Comment'
+import QuestionDetail from '@/components/organisms/QuestionDetail'
+import GET_QUESTION from '@/helpers/graphql/queries/get_question'
+import { loadingScreenShow } from '@/helpers/loaderSpinnerHelper'
+import { errorNotify } from '@/helpers/toast'
+import { useQuery } from '@apollo/client'
+import { useRouter } from 'next/router'
+import { Fragment, useEffect, useState } from 'react'
+import { FilterType } from '../../../index'
+import { AnswerEditType, QuestionType } from '@/pages/questions/[slug]'
+
+type RefetchType = {
+    slug?: string
+    shouldAddViewCount: boolean
+    answerSort?: { column: string; order: string }[]
+}
+
+const QuestionDetailPage = () => {
+    const router = useRouter()
+    const [comment, setComment] = useState(false)
+    const [selectedAnswerFilter, setSelectedAnswerFilter] = useState('Highest Score')
+
+    const [answer, setAnswer] = useState<AnswerEditType>({ id: null, content: null })
+
+    const query = router.query
+
+    const { data, loading, error, refetch } = useQuery<any, RefetchType>(GET_QUESTION, {
+        variables: {
+            slug: String(query['question_slug']),
+            shouldAddViewCount: true,
+            answerSort: [{ column: 'VOTES', order: 'DESC' }],
+        },
+        fetchPolicy: 'network-only',
+    })
+
+    if (loading) return loadingScreenShow()
+    else if (error) return errorNotify(`Error! ${error}`)
+    const question: QuestionType = {
+        ...data.question,
+    }
```

Comment:
```suggestion
    const { data: {question}, loading, error, refetch } = useQuery<any, RefetchType>(GET_QUESTION, {
        variables: {
            slug: String(query['question_slug']),
            shouldAddViewCount: true,
            answerSort: [{ column: 'VOTES', order: 'DESC' }],
        },
        fetchPolicy: 'network-only',
    })

    if (loading) return loadingScreenShow()
    else if (error) return errorNotify(`Error! ${error}`)
```
If above is not working, you can use below
`const question: QuestionType = data.question;`